source/cellular_bg96.c

```diff
@@ -356,3 +363,76 @@ CellularError_t Cellular_ModuleEnableUrc( CellularContext_t * pContext )
 }
 
 /*-----------------------------------------------------------*/
+
+CellularError_t Cellular_CreateMqttSocket( CellularContext_t * pContext, int8_t* pMqttIndex)

```

Comment:
Are these functions not part of the API? At least some of them get called from the cellular_mqtt_wrapper.c

Reply:
Will move functionality into bg96_cellular_api